### DISCUSSION TITLE
第4节课笔记(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#349] [笔记]
> 第4节课笔记(10班)，请在下方提交👇

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135482950?spm=1001.2014.3001.5501

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec4

**Comment by admlrz** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135648392

**Comment by lishuang70345** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135514838

**Comment by Tian-CASIA** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135554798

**Comment by adela778** [comment]
https://blog.csdn.net/Adela0546/article/details/135855484
